workspace/services/iooperations.go

```diff
@@ -0,0 +1,65 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/gablesiak/datatypes"
+	"github.com/google/uuid"
+)
+
+func OpenFile() []byte {
+	inputFile := "input.json"
+
+	inputByte, err := ioutil.ReadFile(inputFile)
+
+	if err != nil {
+		fmt.Print(err)
+		os.Exit(1)
+	}
+
+	return inputByte
+}
+
+
+func CreateInputStruct() []datatypes.InputUserData{
+	inputByte := OpenFile()
+
+	inputStruct := []datatypes.InputUserData{}
+
+	err := json.Unmarshal([]byte(inputByte), &inputStruct)
+
+	if err != nil {
+		fmt.Print(err)
+		os.Exit(1)
+	}
+
+	return inputStruct
+
+}
+
+
+func SaveUser(newUser datatypes.InputUserData){
+	ValidateData(newUser)
+	userEntry :=TransformData(newUser)
+	GenerateOutput(userEntry)
+}
+
+func GenerateOutput(outputData datatypes.OutputUserData){
```

Comment:
_ValidateData_, _TransformData_ and _GenerateOutput_ are internal functions. Those should start with lowercase.